llvm/lib/DebugInfo/DWARF/DWARFVerifier.cpp

```diff
@@ -1512,7 +1512,8 @@ DWARFVerifier::verifyNameIndexAbbrevs(const DWARFDebugNames::NameIndex &NI) {
       NumErrors += verifyNameIndexAttribute(NI, Abbrev, AttrEnc);
     }
 
-    if (NI.getCUCount() > 1 && !Attributes.count(dwarf::DW_IDX_compile_unit)) {
+    if (NI.getCUCount() > 1 && !Attributes.count(dwarf::DW_IDX_compile_unit) &&
+        !Attributes.count(dwarf::DW_IDX_type_unit)) {
       ErrorCategory.Report("Abbreviation contains no attribute", [&]() {
         error() << formatv("NameIndex @ {0:x}: Indexing multiple compile units "
                            "and abbreviation {1:x} has no {2} attribute.\n",
```

Comment:
```suggestion
                           "and abbreviation {1:x} has no type or compile unit attribute.\n",
```

.. or something like that. Basically to have the error message say that we're not looking for just the compile unit attr.

Reply:
This error message is only for `DW_IDX_comp_unit` missing when we have multiple compile units. This code change stops this error being emitted if we have a name entry that only has a `DW_IDX_type_unit`, which is ok.